pkg/views/workspace/selection/view.go

```diff
@@ -161,5 +189,31 @@ func (d ItemDelegate[T]) Spacing() int {
 }
 
 func (d ItemDelegate[T]) Update(msg tea.Msg, m *list.Model) tea.Cmd {
+	i, ok := m.SelectedItem().(item[T])
+	if !ok {
+		return nil
+	}
+
+	m.StatusMessageLifetime = time.Millisecond * 2000
+
+	var title string
+	switch msg := msg.(type) {
+	case tea.KeyMsg:
+		switch {
+		case tea.Key(msg).String() == "x":
+			if strings.Contains(i.title, "Delete: ") {
```

Comment:
Couldn't we use `if i.markForDeletion`?

Reply:
Yes. It will be a better approach. Changed this